docs/blog/es-tooling.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: ES Tooling - community lead e18e-focused tooling
+title: ES Tooling – community-led e18e-focused tooling
```

Comment:
I don't think we need to change this, except "community-led" which do seem more correct

Reply:
Here you have a hyphen instead of an n–dash. The hyphen is not correct.